zuul.d/nodesets.yaml

```diff
@@ -31,16 +31,16 @@
 
 # Ansible cloud appliance nodesets
 - nodeset:
-    name: vmware-vcsa-6.7.0-python36
```

Comment:
I don't think we can do this, because we want to support multiple versions of vsca for stable branches.

Reply:
As I explain in the description, the vcsa term just brings confusion without any real value. So I would still prefer something like `vmware-vcenter-6.7.0-python36`.

When the ESXi test will be in place, the name will be seriously over complicated. e.g: 
vmware-1vcenter-6.7.0U3b-2esxi-6.7.0U3-python36

I would prefer we keep rather agnostic nodeset names. Also do we really need the Python version there?:

- vmware-1vcenter-2esxi-6.7
- vmware-1vcenter-1esxi-6.5

Reply:
I think we can rename the job, but the nodeset maybe not. Otherwise, we have no way to bring online a different version, which is something we need to do to support stable branches still.

So the job name is fine, I think. We also want python36, as we plan on adding python27, python35, python37 and python38 jobs.  In zuul, each job still needs a unique name.

Reply:
> I think we can rename the job, but the nodeset maybe not. Otherwise, we have no way to bring online a different version, which is something we need to do to support stable branches still.

:thinking: We can still us a suffix, like `lastest` for 6.7.0U3, `legacy` for 6.5. This being said, we may want to mix several versions at some point:

- vmware-vcenter-only-latest
- vmware-vcenter-only-legacy
- vmware-vcenter-1esxi-legacy

> So the job name is fine, I think. We also want python36, as we plan on adding python27, python35, python37 and python38 jobs. In zuul, each job still needs a unique name.

Do we really need the python version in the nodeset. The version depends on the controller, and this node may provider some other Python versions.

Reply:
Well, job name is fine, we have the ability to create variants in zuul. So we don't need latest or 6.5 in the name, just the nodeset.

The python version in question, is controller side. For that we need to support 5 python versions, this is what we do today in network

Reply:
So, for a single branch, I'd expect us to only run a single version of vcenter. And for older branches, maybe another. The job name is the same, but we'd have a different nodeset. This is called a job variant in zuul.